scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts

```diff
@@ -22,15 +23,18 @@ async function attemptLiquidation(
   targetAddresses: string[]
 ) {
   try {
+    googleCloudLog(LogSeverity.INFO, `Attempting to liquidate ${targetAddresses} via ${liquidator.address}`);
     await liquidator.connect(signer).initFlash({
       accounts: targetAddresses,
       pairToken: daiPool.tokenAddress,
       poolFee: daiPool.poolFee
     });
-    console.log(`Successfully liquidated ${targetAddresses}`);
+    googleCloudLog(LogSeverity.INFO, `Successfully liquidated ${targetAddresses} via ${liquidator.address}`);
   } catch (e) {
-    console.log(`Failed to liquidate ${targetAddresses}`);
-    console.log(e.message);
+    googleCloudLog(
+      LogSeverity.ALERT,
```

Comment:
Does this actually alert someone right now?

Reply:
not yet, but it means we can construct a Policy that alerts if there are more than X number of `ALERT` level logs within a period of time

Reply:
Nice